Update network-watcher-nsg-flow-logging-powershell.md

jrudley opened the pull request:
There is no space on row 47 between nsgRG-Name The Name parameter is being treated as a string and breaking the cmdlet example. I added a space to fix the example

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @dominicbetts, has been notified to review your proposed change.

dominicbetts commented:
@4c74356b41 - thanks for pointing this out. I'll get this change merged.
@tysonn - please merge.

jrudley committed:
Update network-watcher-nsg-flow-logging-powershell.md

There is no space on row 47 between nsgRG-Name The Name parameter is being treated as text and breaking the cmdlet example. I added a space to fix the example
